build_tools/python/benchmark_suites/iree/utils.py

```diff
@@ -15,6 +15,7 @@ def generate_e2e_model_run_configs(
         iree_definitions.ModuleGenerationConfig],
     module_execution_configs: Sequence[iree_definitions.ModuleExecutionConfig],
     device_specs: Sequence[common_definitions.DeviceSpec],
+    tags: Sequence[str] = [],
```

Comment:
It's suggested not to use a mutable object as the initializer. Maybe

```suggestion
    tags: Optional[Sequence[str]] = None,
```

Reply:
More detail: https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/

Reply:
Thanks. Added.